tracker/tracker-assist/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@openreplay/tracker-assist",
   "description": "Tracker plugin for screen assistance through the WebRTC",
-  "version": "4.1.1",
+  "version": "4.1.2",
```

Comment:
one last piece, can you bump the tracker version in peer deps to >=4.1.6 since we're using methods from there

Reply:
Fixed it